apps/full-site-editing/full-site-editing-plugin/blog-posts-block/newspack-homepage-articles/blocks/homepage-articles/index.js

```diff
@@ -42,14 +42,18 @@ export const settings = {
 	attributes,
 	category,
 	keywords: [
-		__( 'posts', 'newspack-blocks' ),
-		__( 'articles', 'newspack-blocks' ),
-		__( 'latest', 'newspack-blocks' ),
+		__( 'posts', 'full-site-editing' ),
+		__( 'articles', 'full-site-editing' ),
+		__( 'latest', 'full-site-editing' ),
```

Comment:
Adding context here (`_x( 'posts',  'block keyword', 'full-site-editing' )`) could be helpful for translators.

Reply:
This should be addressed in Newspack code 👍